test/user-address-test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from "chai";
+import { prisma } from "../src/setup-db";
+
+describe("Address Tests", function () {
+  it("Added two addresses to one user", async function () {
+    const userDB = await prisma.user.create({
+      data: {
+        name: "User Test",
+        email: "test@example.com",
+        password: "123abc",
+        birthDate: "01-01-2000",
+        address: {
+          create: [
+            {
+              zipCode: 12345678,
+              street: "Street 1",
+              streetNumber: 1,
+              city: "City 1",
+              state: "State 1"
+            },
+            {
+              zipCode: 12345678,
+              street: "Street 2",
+              streetNumber: 2,
+              city: "City 2",
+              state: "State 2"
+            }
+          ]
+        }
+      }
+    });
+
+    const address = await prisma.address.findMany({
```

Comment:
```suggestion
    const addresses = await prisma.address.findMany({
```
se vc está buscando uma lista, o nome da variável precisa ficar no plural para estar de acordo